chapters/02-estrutura-do-programa.md

```diff
@@ -266,43 +266,39 @@ while (number <= 12) {
 //   … etcetera
 ```
 
-Uma declaração iniciando com a palavra `while` cria um loop. A palavra `while` é seguida por uma expressão em parênteses e então uma declaração, como um `if`. O loop age continuando a executar a declaração enquanto a expressão produzir um valor que é `true`, quando convertido para o tipo booleano.
+Uma declaração que inicia com a palavra-chave `while` cria um loop. A palavra `while` é acompanhada por uma expressão entre parênteses e seguida por uma declaração, similar ao `if`. O loop continua executando a declaração enquanto a expressão produzir um valor que, após convertido para o tipo Booleano, seja `true`.
 
+Nesse loop, queremos imprimir o número atual e somar dois em nossa variável. Sempre que precisarmos executar múltiplas declarações dentro de um loop, nós as envolvemos com chaves (`{` e `}`). As chaves, para declarações, são similares aos parênteses para as expressões, agrupando e fazendo com que sejam tratadas como uma única declaração. Uma sequência de declarações envolvidas por chaves é chamada de _bloco_.
```

Comment:
Sobre os dois pontos, não necessariamente precisamos ficar presos a estrutura gramatical do original, principalmente porque o português é uma língua muito mais complexa que o inglês.  Nesse caso, o dois pontos foi trocado por uma vírgula, o que não muda a tradução e o significado. 

Sobre o itálico, o original usa itálico para dar ênfase ao introduzir alguns termos pela primeira vez e em algumas palavras específicas que precisam de destaque, o que faz sentido. No nosso caso, usamos o itálico também para dar destaque aos termos em inglês. Um não exclui o outro.